db/mongodb.go

```diff
@@ -226,3 +226,9 @@ func (m *MongoDB) GetCompany(id string) (string, error) {
 	}
 	return string(b), nil
 }
+
+func (m *MongoDB) ExtraIndexes(idxs []string) error {
+	log.Output(1, fmt.Sprintf("indexes: %s", strings.Join(idxs, ", ")))
+	log.Output(1, fmt.Sprintf("has not yet been implemented"))
+	return nil
```

Comment:
Isso deveria ser um erro:

```suggestion
	return fmt.Errorf("extra-indexes not implemented (yet)")
```